enhancements/sig-policy/98-long-term-compliance-history/README.md

```diff
@@ -0,0 +1,496 @@
+    "message": "configmaps [app-data] not found in namespace default",
+    "timestamp": "2023-07-19T18:25:43.511Z",
+    // Optional
+    "metadata": {}
+  }
+}
+```
+
+If the compliance event is recorded correctly, an HTTP 201 status code is returned.
+
+##### Viewing Compliance Events
+
+Another HTTP endpoint would be added at `/api/v1/compliance-events` that would accept a `GET` request. This would be
+responsible for exposing the compliance events and performing the correct authorization. The authorization would require
+`get` permissions on the cluster's associated `ManagedCluster` object to be able to list the compliance events for the
+cluster. For performance reasons, this is preferred to performing a `SubjectAccessReview` on every cluster namespace for
+a user per request. We can utilize the
+[github.com/stolostron/rbac-api-utils](https://github.com/stolostron/rbac-api-utils/blob/159deac7d398c3470ae730c3fbb8ffbbcca1ac5a/pkg/rbac/rbac.go#L187)
+library to do this.
+
+The following optional query parameters would be accepted on this HTTP endpoint:
+
+- cluster - the name of the cluster to filter on. Multiple can be provided with `?cluster=local-cluster&cluster=managed`
+  syntax.
+- parent_policy_name - the name of the parent policy (i.e. a `Policy`) of the policy (e.g. a `ConfigurationPolicy`) that
+  generated the compliance event to filter on.
+- parent_policy_category - a category name of the parent policy (e.g. `CM Configuration Management`) to filter on.
+  Multiple can be provided with
+  `?parent_policy_category=CM%20Configuration%20Management&parent_policy_category=CF%20Configuration` syntax.
+- parent_policy_control - a control name of the parent policy (e.g. `Baseline Configuration`) to filter on. Multiple can
+  be provided with `?parent_policy_control=Baseline%20Configuration&parent_policy_control=Other%20Configuration` syntax.
+- parent_policy_standard - a standard name of the parent policy (e.g. `NIST SP 800-53`) to filter on. Multiple can be
+  provided with `?parent_policy_standard=NIST%20SP%20800-53&parent_policy_standard=NIST%20SP%20231-41` syntax.
+- policy - the name of the policy (e.g. a `ConfigurationPolicy`) that generated the compliance event to filter on.
+- policy_kind - the kind of the policy (e.g. `ConfigurationPolicy`) that generated the compliance event to filter on.
+- policy_api_group - the `apiGroup` of the policy (e.g. `policy.open-cluster-management.io`) that generated the
+  compliance event to filter on.
+- include_spec - a boolean to include the policy's spec in the return value. This is `false` by default.
+- before - an ISO-8601 timestamp to indicate only compliance events before this time should be shown.
+- after - an ISO-8601 timestamp to indicate only compliance events after this time should be shown.
+- page - the page number in the query. This defaults to 1.
+- per_page - the number of compliance events returned per page. This defaults to 20 and maxes out at 100.
```

Comment:
How about add `search` query? search with **words**

Reply:
Good idea. I'll add the `message_includes` query parameter.